Exiled.Events/Handlers/Server.cs

```diff
@@ -17,7 +17,7 @@ namespace Exiled.Events.Handlers
     /// <summary>
     /// Server related events.
     /// </summary>
-    public class Server
+    public static class Server
```

Comment:
For now, the class will remain non-static, but if in the 2.0.0 full release we won't change the event system again (like making it per-plugin instead of global) we could just set all events classes to static.